pkg/agent/metrics/prometheus.go

```diff
@@ -67,6 +67,54 @@ var (
 		Help:           "Flow count for each OVS flow table. The TableID is used as a label.",
 		StabilityLevel: metrics.STABLE,
 	}, []string{"table_id"})
+
+	OVSFlowAddErrorCount = metrics.NewCounter(
```

Comment:
Do you think we have one metric `OVSFlowOpsErrorCount` and have add, modify and delete as labels? It seems to be possible for summary duration metrics: https://github.com/prometheus/client_python/issues/82

Reply:
+1

Reply:
Sure, I'll take a look at it.

Reply:
I did some research and think that it's good to use NewConterVec for ErrorCount and NewSummaryVec for Duration. 
But please kindly let me know if SummaryOpts is better than them. 
I think it's good because constLabels are static and put to all measured metrics.

Hence I try to define the metrics using the below label.
- antrea_agent_ovs_flow_ops_error_count - differentiate operations: operation="add|modify|delete"
![image](https://user-images.githubusercontent.com/17121369/85953183-81696100-b9a9-11ea-9fa5-19f632bfec71.png)

- antrea_agent_ovs_flow_ops_duration_milliseconds - differentiate operations: operation="add|modify|delete"
![image](https://user-images.githubusercontent.com/17121369/85953195-8fb77d00-b9a9-11ea-88d3-6491a3ba092f.png)